addons/mail/models/mail_activity.py

```diff
@@ -12,10 +12,15 @@
 
 from odoo.tools.misc import clean_context
 from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
+from odoo.addons.base.models.res_partner import _tz_get
 
 _logger = logging.getLogger(__name__)
 
 
+def tz2int(tz):
+    return int(datetime.now(pytz.timezone(tz)).strftime('%z'))
```

Comment:
Nope ?

Reply:
In this case it's not needed as we sure for tz value, but if we move this to odoo.tools, then it must be handle empty values too